src/index.html

```diff
@@ -9,6 +9,10 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="clock">
+      <div class="clock__circle"></div>
```

Comment:
maybe will be better remove div with  class `clock__circle` and show circle using `:before or :after`?